RssReader/logs.py

```diff
@@ -0,0 +1,94 @@
+"""This module creates logs, writes logs, reads logs."""
+
+import logging
+import os
+
+this_directory = os.path.abspath(os.path.dirname(__file__))
+
+journalLog = 'logJournal.log'
+directory = os.path.join(this_directory, journalLog)
+
+args = '%(asctime)s - %(levelname)s - %(message)s'
+logging.basicConfig(filename=directory, format=args, level=logging.INFO, filemode='a+')
+
+
+def new_session():
```

Comment:
Why is this necessery to have such functions? Is this better than just to pass some text to be logged into a `logging.info()`?

Reply:
I put these functions into separate logs.py module and used them instead of simple form logging.info() since they are repeated in other modules for several times. I thought it would be more beautiful to have a line like logs.new_session() in the code, than logging.info('A very long comment'). Moreover such implementation allows me to change them very easy and fast within one logs.py module, instead of searching them all over the program.